app/ui_screen.js

```diff
@@ -56,6 +131,11 @@ const UI = {
 
     connect() {
         console.log('connect')
+        const details = {
+            left: window.screenLeft,
+            top: window.screenTop
+        }
+        console.log(details)
```

Comment:
ensure we either remove console.log used for debugging, use the Log.Debug function if we should keep the logging info. I do see other console.log statements that were not part of this MR, I do need to go back and do some cleanup on the multi-monitor stuff, I likely missed cleaning up my own.

Reply:
Done, I also removed all the other console.logs in ui_screen.js